src/Requests.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+async function makeAccountPostCall(person){
+   try {
+   const response = await axios.post('http://localhost:5000/users', person);
+   return response;
+   }
+   catch (error) {
+   console.log(error);
+   return false;
+   }
+}
+
+export function addAccount(person) {
+   makeAccountPostCall(person).then( result => {
+   if (result && result.status === 201)
+      console.log('CreateAccount: Success!');
+   });  
+}
+
+async function makeLoginPostCall(person){
+   try {
+   const response = await axios.post('http://localhost:5000/login', person);
+   return response;
+   }
+   catch (error) {
+   console.log(error);
+   return false;
+   }
+}
+
+export function checkCredentials(person) {
+   makeLoginPostCall(person).then( result => {
+   if (result && result.status === 200)
+      console.log('Login: Success!');
+   else
+      console.log('Login: Failure.');
+   });  
+}
```

Comment:
This call from the backend returns the user, you have to use it to populate the user info.

Reply:
Also, if it is not too much work, could you also add a conditional check or a login button on the header? Because we've removed the `welcome.js` page in favor of defaulting it to the home page. 

Thanks.